tests/functional/spec/features/routing/or.spec.js

```diff
@@ -0,0 +1,38 @@
+import FirstNumberQuestionPage from "../../../generated_pages/new_routing_or/number-question-1.page";
+import SecondNumberQuestionPage from "../../../generated_pages/new_routing_or/number-question-2.page";
+import CorrectAnswerPage from "../../../generated_pages/new_routing_or/correct-answer.page";
+import IncorrectAnswerPage from "../../../generated_pages/new_routing_or/incorrect-answer.page";
+
+describe("Feature: Routing - OR Operator", () => {
+  describe("Equals", () => {
+    describe("Given I start the or operator routing survey", () => {
+      beforeEach(() => {
+        browser.openQuestionnaire("test_new_routing_or.json");
+      });
+
+      it("When I enter 123 and 321, Then I should be routed to the correct page", () => {
```

Comment:
I changed these based on @MebinAbraham comments above to be more explicit, previously used things like `When I enter both answers incorrectly` etc. So not sure which approach to take here now

Reply:
The problem is I have no idea if 123 and 321 is correct from this statement, that is the most important thing, not what the number is, in this number it's arbitrary, just needs to be right/expected or wrong/not expected. Now because Mebin will argue till my bedtime, lets find a middle ground. 
"When I enter 123 and 321 which are both correct, Then I should be routed to the correct page". Anything like that works. "both true", "both expected"

Reply:
alright cool i will update it 👍

Reply:
Have updated these now